tests/behat/features/web/translation/translate_project.feature

```diff
@@ -7,29 +7,111 @@ Feature: Project title, description and credits should be translatable via a but
+    Then the element "#program-translation-button" should be visible
+    And the element "#name" should be visible
+    And the element "#credits" should be visible
+
+  Scenario: Translate button should translate title, description, and credit when available
+    Given I am on "/app/project/4"
+    And I wait for the page to be loaded
+    Then the element "#program-translation-button" should exist
+    When I click "#program-translation-button"
+    And I wait for AJAX to finish
+    Then the element "#remove-program-translation-button" should be visible
+    And the element "#name-translation" should be visible
+    And the "#name-translation" element should contain "translated project4"
+    And the element "#description-translation" should be visible
+    And the "#description-translation" element should contain "translated mydescription"
+    And the element "#credits-translation" should be visible
+    And the "#credits-translation" element should contain "translated mycredit"
+    When I click "#remove-program-translation-button"
+    Then the element "#program-translation-button" should be visible
+    And the element "#name" should be visible
+    And the element "#description" should be visible
+    And the element "#credits" should be visible
+
+  Scenario: Translate button should translate title, description, and credit only once
+    Given I am on "/app/project/1"
+    And I wait for the page to be loaded
+    Then the element "#program-translation-button" should exist
+    When I click "#program-translation-button"
+    And I wait for AJAX to finish
+    Then the element "#remove-program-translation-button" should be visible
+    And the element "#name-translation" should be visible
+    When I click "#remove-program-translation-button"
+    Then the element "#program-translation-button" should be visible
+    And the element "#name" should be visible
+    When I click "#program-translation-button"
+    Then the element "#remove-program-translation-button" should be visible
+    And the element "#name-translation" should be visible
+
+  Scenario: Translation should show translated by line
+    Given I am on "/app/project/4"
+    And I wait for the page to be loaded
+    Then the element "#program-translation-button" should exist
+    When I click "#program-translation-button"
+    And I wait for AJAX to finish
+    Then the element "#credits-translation-wrapper" should be visible
+    And the "#program-translation-before-languages" element should contain "Translated by iTranslate from"
+    And the "#program-translation-first-language" element should contain "English"
+    And the "#program-translation-between-languages" element should contain "to"
+    And the "#program-translation-second-language" element should contain "English"
+    And the "#program-translation-after-languages" element should contain ""
+  
+  Scenario: Translation should not change description and credits when description and credits are not available

```

Comment:
i think you can merge this test case in the first 4s

Reply:
Moved this test's checks to the first test in this file